shell/app/common/components/tags/index.scss

```diff
@@ -76,7 +76,11 @@
   }
 }
 
-.tags-row-tooltip {
+body .tags-row-tooltip {
```

Comment:
will it be wrong if I remove the `body`?

Reply:
To increase the weight of the style, if removed it will be overwritten by the global style